README.md

```diff
@@ -1,11 +1,8 @@
-# Mcap file format ![](https://img.shields.io/badge/alpha-orange)
+# MCAP file format ![](https://img.shields.io/badge/alpha-orange)
 
-Mcap is a modular container format for recording pub/sub messages with arbitrary message serialization.
-It is primarily intended for use in robotics applications, and works well under various workloads, resource constraints, and durability requirements.
+MCAP is a modular container format for recording pub/sub messages with arbitrary message serialization. It is primarily intended for use in robotics applications, and works well under various workloads, resource constraints, and durability requirements.
 
-## Documentation
-
-- [File format specification](./docs/specification)
+The file format specification is documented [here](./docs/specification).
```

Comment:
I think this is harder to notice than before. The prior way made it a bullet item which is easy to quickly read.

Reply:
agree I prefer how it was

Reply:
I don't feel strongly either way. A bulleted list with only one item is strange. This will probably evolve in the future and get rewritten as we really flesh out our documentation.

Reply:
I think we should add john's original whitepaper to this repo and add it to this list under "motivation"

Reply:
Reverted to separate section.

Added John's whitepaper PDF to the repo and to this bullet point list.